cli.sh

```diff
@@ -1066,12 +1065,11 @@ list_shared_hook_repos() {
             SHARED_REPOS_LIST=$(grep -E "^[^#].+$" <"$(pwd)/.githooks/.shared")
 
             echo "$SHARED_REPOS_LIST" | while read -r LIST_ITEM; do
-                NORMALIZED_NAME=$(echo "$LIST_ITEM" |
-                    sed -E "s#.*[:/](.+/.+)\\.git#\\1#" |
-                    sed -E "s/[^a-zA-Z0-9]/_/g")
 
-                if [ -d ~/.githooks/shared/"$NORMALIZED_NAME"/.git ]; then
-                    if [ "$(cd ~/.githooks/shared/"$NORMALIZED_NAME" && git config --get remote.origin.url)" = "$LIST_ITEM" ]; then
+                set_shared_root "$LIST_ITEM"
+
+                if [ -d "$SHARED_ROOT/.git" ]; then
+                    if [ "$(cd "$SHARED_ROOT/.git" && git config --get remote.origin.url)" = "$LIST_ITEM" ]; then
```

Comment:
Same as above, `cd` into the repo root instead of `.git` please if that works.

Reply:
ah jeah!